saleor/checkout/tests/test_tasks.py

```diff
@@ -619,6 +636,7 @@ def test_automatic_checkout_completion_task_transaction_flow(
     # then
     assert not Checkout.objects.filter(pk=checkout_pk).exists()
     order = Order.objects.filter(checkout_token=checkout_pk).first()
+    assert order.charge_status
```

Comment:
Shouldn't be `assert order.charge_status == <expected-charge-status>? Right now it will always be `true` as default value is not `None`

Reply:
And also, assert from line 640 should be called first 🤔